content/en/news/2019/announcing-1.4/upgrade-notes/index.md

```diff
@@ -21,3 +21,18 @@ If you depend on this behavior, there are a few options:
 * Specify the environment variable `PILOT_BLOCK_HTTP_ON_443=false` to the Pilot deployment. Note: this may be removed in future releases.
 
 See [Protocol Selection](/docs/ops/traffic-management/protocol-selection/) for more information about specifying the protocol of a port
+
+## Configuration Management
```

Comment:
```suggestion
## Configuration management
```

Reply:
I thought we are using capitals for all words in a title? like above Traffic Management

Reply:
Nope, only for doc titles. Headings use first capital only.

https://istio.io/about/contribute/style-guide/#use-title-capitalization-for-title-front-matter

Reply:
ok. Done. Also changed the traffic management heading.